java/core/src/main/java/sleeper/core/partition/PartitionsFromSplitPoints.java

```diff
@@ -109,18 +108,18 @@ private List<Partition.Builder> addLayer(List<Partition.Builder> partitionsInLay
                 ranges.add(rangeForDim0);
                 Region region = new Region(ranges);
                 String id = UUID.randomUUID().toString();
+                List<String> childPartitionIds = List.of(leftPartition.getId(), rightPartition.getId());
                 Partition.Builder parent = Partition.builder()
                         .id(id)
                         .parentPartitionId(null)
-                        .childPartitionIds(Arrays.asList(leftPartition.getId(), rightPartition.getId()))
+                        .childPartitionIds(childPartitionIds)
                         .leafPartition(false)
                         .dimension(0)
                         .rowKeyTypes(schema.getRowKeyTypes())
                         .region(region);
-
                 leftPartition.parentPartitionId(id);
                 rightPartition.parentPartitionId(id);
-
+                LOGGER.debug("Created parent partition " + id + " joining partitions " + childPartitionIds);
```

Comment:
This should use ("Created... {} joining partitions {}", id, childPartitionIds) rather than performing the string concatenation itself.